### ISSUE TITLE
Can't use with .chs files

### ISSUE THREAD

**Issue by falsandtru** [haskell/haskell-ide-engine#1178]
HIE throws an error with https://github.com/crclark/foundationdb-haskell/blob/master/src/FoundationDB/Error/Internal.hs#L13 because HIE can't parse https://github.com/crclark/foundationdb-haskell/blob/master/src/FoundationDB/Internal/Bindings.chs. Can you support .chs files?

**Comment by mpickering** [comment]
What is the actual error you have encountered?

**Comment by falsandtru** [comment]
```
Could not find module ‘FoundationDB.Internal.Bindings’
Use -v to see a list of the files searched for.
```

**Comment by mpickering** [comment]
It seems the problem here is that the immediate when the cradle is set up the chs are not preprocessed. Does anyone know how this currently works for alex/happy files?

**Comment by dsturnbull** [comment]
I don't think it does work quite right with alex/happy files, at least the current build as of today.

I have a sub-library defined that builds an alex module and a happy module. The cradle log shows it's unable to find the alex module.

As a workaround, I've further split my alex+happy-using sublibrary into one that only has the alex module and another new sublibrary with the rest of the stuff that uses it, including the happy module. The cradle seems perfectly able to build as it seems it can find things as long as they aren't local imports.
